### DISCUSSION TITLE
Fetching Images from GAP8

### DISCUSSION THREAD

**Discussion by yanrui89** [bitcraze/discussions#524] [Q&A]
Appreciate help on this. I am trying to fetch the camera images from GAP8 to my local laptop. So I ran the opencv-viewer.py file (https://www.bitcraze.io/documentation/repository/aideck-gap8-examples/master/test-functions/wifi-streamer/) and indeed managed to achieve a rate of 7Hz of images fetched to my laptop. We achieved this with a router that is able to stream 1000Mbps (2.4Ghz). We thus proceeded to collect images from multiple drones in parallel (7 in total) by spawning 7 processes, and each process connects to each of the drone’s ipaddress. However, when we tried to stream all 7 drones’ images simultaneously, the rate of images streamed from each drone drop to between 3-5Hz. However, our router should be more than capable of handling the rate of data transmission. Could we seek help to ask if anyone faced similar issues or have any solutions to streaming multiple drones’ images at a healthy rate through a single router?

**Comment by pladosz** [comment]
Hi,
In our experience, wi-fi modules on the AI-deck are very susceptible to interference which can result in slower transmission rate or even full disconnects. I suspect in your case they are just interfering with each other.

**Reply by yanrui89** [reply]
Thank you so much for your reply! I was wondering if you have any solutions to the interference problem? Would you be able to provide an estimate of streaming rates you managed to achieve with your drones? Thank you!

**Reply by pladosz** [reply]
The fastest we have seen was about 8.5 Hz.

I don't have any clear solution. My only suggestion would be to try to have two or three wi-fi networks each on different wi-fi channels, each connected to 2-3 crazyflies. I am not really sure it would solve your problem, but that's the best I can think of.

**Reply by yanrui89** [reply]
Thank you! May I also ask if you know if Bluetooth can be deactivated? Is so, would you know how?

**Reply by krichardsson** [reply]
@yanrui89 Please start a new discussion for your Bluetooth question as it is too far away from the title of this discussion
